src/imager.jsx

```diff
@@ -1,7 +1,10 @@
 'use strict';
 
+var isBrowser = (typeof window !== 'undefined');
+
 var React = require('react');
-var Imager = require('imager.js');
+
+var Imager = isBrowser ? require('imager.js') : null;
```

Comment:
I would rather use the `browser` field of `package.json` to shim imager (or not).

Server-side, the `onResize` would be always falsy, and the verification would be related to `imgr.onResize` rather than `onResize` directly.

The code remains equal both for the browser and the server, and the behaviour is only driven by what imager.js is supposed to be.